tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist",
+    "target": "ESNext",
+    "moduleResolution": "node",
```

Comment:
can we add another compiler option: `"module": "CommonJS",` trying to run the build output in dist with node gives the error:
```
SyntaxError: Cannot use import statement outside a module 
```

Reply:
I don't think that would be the solution to the problem, because CommonJS  module system only allows `require` so I believe we would get the same error. The problem most likely lies in the `"target": "ESNext"` option because in ES6 `import` statements were added and so it thinks we can use those and it is not converting them.

Reply:
hmm i checked out your branch and was able to resolve this issue by doing what i mentioned. looking online for the error message + typescript yields some other answers though so if you'd prefer a different method to resolve this, lmk what it is